sparrow-py/pysrc/sparrow_py/_timestream.py

```diff
@@ -700,7 +726,7 @@ def lookup(self, key: Union[Timestream, Literal]) -> Timestream:
         Parameters
         ----------
         key : Union[Timestream, Literal]
-            The foreign key to lookup. 
+            The foreign key to lookup.
```

Comment:
Remind me the lint command?

Reply:
This was just vscode. I think `nox -s fix_lint` should reformat though. Or you can run `black` directly.